### DISCUSSION TITLE
회원가입 뷰 스케치

### DISCUSSION THREAD

**Discussion by joonBaek12** [TeamPophory/pophory-iOS#3] [View Sketch]
![UI SKETCH-7](https://github.com/TeamPophory/pophory-iOS/assets/97212841/97d4fecb-5065-48f5-b196-f76aa3f2f2b7)
![UI SKETCH-8](https://github.com/TeamPophory/pophory-iOS/assets/97212841/865675ff-09ed-422e-a68c-5ee70ad02950)
![UI SKETCH-9](https://github.com/TeamPophory/pophory-iOS/assets/97212841/32ff9e17-fa97-4e1c-9613-9906a7c21495)

네비바랑 볼드처리된 label, 보라색 버튼은 제가 분기처리후 익스텐션/Factory로 만들어놓을게요!

**Comment by dannaward** [comment]
<img width="539" alt="image" src="https://github.com/TeamPophory/pophory-iOS/assets/71601985/38185b59-a6ef-4f44-8394-df8df28cc0e4">

앨범 커버 선택하는 이 부분 나중에 몇 개 더 추가될지 모르는데 아싸리 스크롤 대응도 해놓을 겸 컬렉션 뷰로 하시는 거 어떻슴까

**Reply by joonBaek12** [reply]
아 그러네요!! 컬렉션뷰로 수정하겠습니다 ㅎㅎ
